ccm_web/client/src/pages/LaunchError.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { makeStyles, Typography } from '@material-ui/core'
+
+import InlineErrorAlert from '../components/InlineErrorAlert'
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    margin: theme.spacing(2)
+  }
+}))
+
+export default function LaunchError (): JSX.Element {
+  const classes = useStyles()
+
+  return (
+    <div className={classes.container}>
+      <InlineErrorAlert>
+        <Typography variant='subtitle1' component='h1'>Launch Error</Typography>
+        <Typography>
+          The tool could not be launched. This is likely due to settings related to cookies on your Internet browser.
```

Comment:
Should the text be updated as per this https://github.com/tl-its-umich-edu/canvas-course-manager-next/issues/312#issuecomment-1055860902

Reply:
Okay, I'll do this today.